client/components/forms/multi-checkbox/index.jsx

```diff
@@ -1,40 +1,40 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	omit = require( 'lodash/omit' ),
-	debug = require( 'debug' )( 'calypso:forms:multi-checkbox' );
+import React, { Component, PropTypes } from 'react';
+import { omit } from 'lodash';
+import debugFactory from 'debug';
 
-var MultiCheckbox = module.exports = React.createClass({
-	displayName: 'MultiCheckbox',
+const debug = debugFactory( 'calypso:forms:multi-checkbox' );
 
-	propTypes: {
-		defaultChecked: React.PropTypes.array,
-		onChange: React.PropTypes.func,
-		disabled: React.PropTypes.bool
-	},
+export default class MultiCheckbox extends Component {
+	static propTypes = {
+		checked: PropTypes.array,
+		defaultChecked: PropTypes.array,
+		disabled: PropTypes.bool,
+		onChange: PropTypes.func,
+		options: PropTypes.array,
+		name: PropTypes.string,
+	};
 
-	getInitialState: function() {
-		return { initialChecked: this.props.defaultChecked };
-	},
+	static defaultProps = {
```

Comment:
Should we provide defaults for:

- `name`, given the concatenation in `getCheckboxElements`
- `options` given `this.props.options.map` in `getCheckboxElements`